server/multitenancy/tenant_resolver.ts

```diff
@@ -37,10 +37,12 @@ export const GLOBAL_TENANTS: string[] = ['global', GLOBAL_TENANT_SYMBOL];
 export function resolveTenant(
   request: OpenSearchDashboardsRequest,
   username: string,
+  roles: string[] | undefined,
   availabeTenants: any,
   config: SecurityPluginConfigType,
   cookie: SecuritySessionCookie
 ): string | undefined {
+  const DEFAULT_READONLY_ROLES = ['kibana_read_only'];
```

Comment:
here is the same constant defined again. is there a common scope for this constant that could be referenced in these two files?